relay-event-normalization/src/normalize/mod.rs

```diff
@@ -2853,6 +2967,220 @@ mod tests {
         "#);
     }
 
+    #[test]
+    fn test_computed_performance_score() {
+        let json = r#"
+        {
+            "type": "transaction",
+            "timestamp": "2021-04-26T08:00:05+0100",
+            "start_timestamp": "2021-04-26T08:00:00+0100",
+            "measurements": {
+
+                "fid": {"value": 213, "unit": "millisecond"},
+                "fcp": {"value": 1237, "unit": "millisecond"},
+                "lcp": {"value": 6596, "unit": "millisecond"},
+                "cls": {"value": 0.11}
+            },
+            "contexts": {
+                "browser": {
+                    "name": "Chrome",
+                    "version": "120.1.1",
+                    "type": "browser"
+                }
+            }
+        }
+        "#;
+
+        let mut event = Annotated::<Event>::from_json(json).unwrap().0.unwrap();
+
+        let performance_score_config: PerformanceScoreConfig = serde_json::from_value(json!({
+            "profiles": [
+                {
+                    "name": "Desktop",
+                    "score_components": [
+                        {
+                            "measurement": "fcp",
+                            "weight": 0.15,
+                            "p10": 900,
+                            "p50": 1600
+                        },
+                        {
+                            "measurement": "lcp",
+                            "weight": 0.30,
+                            "p10": 1200,
+                            "p50": 2400
+                        },
+                        {
+                            "measurement": "fid",
+                            "weight": 0.30,
+                            "p10": 100,
+                            "p50": 300
+                        },
+                        {
+                            "measurement": "cls",
+                            "weight": 0.25,
+                            "p10": 0.1,
+                            "p50": 0.25
+                        },
+                    ],
```

Comment:
How likely is it that the formula for the score will every involve other measurements or even input from somewhere else in the event, or that it will have other parameters than weight, p10, and p50?

If likely, we could use this as an opportunity to build a fully generic computation config, that can take and combine values from anywhere in the event (similar to our [metrics extraction config](https://github.com/getsentry/relay/blob/b9e9f9377d7e9a4e477a51335e030a46613c2844/relay-dynamic-config/src/metrics.rs#L216)), so that even the formula for the `erf` function would be provided by config.

If unlikely, we can probably simplify this to 

```json
{"fcp": {"weight": 0.15, "p10": 900, "p50": 1600}, "lcp": ...}
```

Reply:
fwiw, this is still semi experimental since we aren't using it in any GA'd product (yet).

> involve other measurements or even input from somewhere else in the event, or that it will have other parameters than weight, p10, and p50?

Very possible, hence why I didn't hardcode it. We've been debating including TTFB for example, that may change and this allows us that flexibility. Input from elsewhere in the event _may_ take into account duration etc. 

>  build a fully generic computation config

Yeah I think at the point in time we know what we want for the next iteration we can make it generic then? I was thinking to avoid anything generic until we've confirm a couple cases.